doc/api/stream.md

```diff
@@ -934,9 +939,8 @@ The Readable stream will properly handle multi-byte characters delivered through
 the stream that would otherwise become improperly decoded if simply pulled from
 the stream as `Buffer` objects.
 
-Encoding can be disabled by calling `readable.setEncoding(null)`. This approach
-is useful when working with binary data or with large multi-byte strings spread
-out over multiple chunks.
+Decoding can be disabled by calling `readable.setEncoding(null)` or
+`readable.setEncoding('buffer')`.
```

Comment:
That’s part of why I wanted to change this paragraph, I hope the result is okay. I’ll gladly take suggestions for improvements!